app/models/profile.rb

```diff
@@ -119,12 +119,10 @@ def in_account(account)
                     benchmark_id: benchmark_id)
   end
 
-  def add_rules_from(profile: nil)
-    new_profile_rules = profile.profile_rules - profile_rules
-    ProfileRule.import!(new_profile_rules.map do |profile_rule|
-      new_profile_rule = profile_rule.dup
-      new_profile_rule.profile = self
-      new_profile_rule
+  def add_rules(ref_ids)
```

Comment:
Explicit naming goes a long way towards readability

```suggestion
  def add_rule_ref_ids(ref_ids)
```

or (my preference)

```suggestion
  def add_rules(ref_ids:)
```

Otherwise, the assumption is that this method is meant to receive Rule objects.